boba-chain-ops/genesis/layer_two_test.go

```diff
@@ -0,0 +1,64 @@
+package genesis_test
+
+import (
+	"context"
+	"encoding/json"
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/bobanetwork/v3-anchorage/boba-bindings/bindings"
+	"github.com/bobanetwork/v3-anchorage/boba-bindings/predeploys"
+	"github.com/bobanetwork/v3-anchorage/boba-chain-ops/deployer"
+	"github.com/bobanetwork/v3-anchorage/boba-chain-ops/genesis"
+	"github.com/ledgerwatch/erigon-lib/common"
+	"github.com/ledgerwatch/erigon/crypto"
+	"github.com/stretchr/testify/require"
+)
+
+var writeFile bool
+
+func init() {
+	flag.BoolVar(&writeFile, "write-file", false, "write the genesis file to disk")
```

Comment:
I'm a bit confused by this part of the test.  Is this a production path? Something that was just used to help debug the test?  When should the user pass this flag?

Reply:
It's from Optimism. I can delete them. :)

Reply:
Removed